website/content/docs/nia/configuration.mdx

```diff
@@ -58,14 +58,14 @@ tls {
   - `ca_cert` - (string) The path to a PEM-encoded certificate authority file used to verify the authenticity of the incoming client connections to the CTS API when `verify_incoming` is set to true. Takes precedence over `ca_path` if both are configured.
   - `ca_path` - (string) The path to a directory of PEM-encoded certificate authority files used to verify the authenticity of the incoming client connections to the CTS API when `verify_incoming` is set to true.
 - `license_path` <EnterpriseAlert inline /> - (string) **Deprecated in CTS 0.6.0 and will be removed in a future release. Use [license block](/docs/nia/configuration#license-1) instead.**  Configures the path to the file containing the license. A license must be set to use enterprise features. You can also set the license by defining the `CONSUL_LICENSE` and `CONSUL_LICENSE_PATH` environment variables. Review [Setting the License](/docs/nia/enterprise/license#setting-the-license) for details.
```

Comment:
Sorry! Just noticed it looks like `/docs/nia/configuration#license-1` might have changed to `/docs/nia/configuration#license`